run_gemini.py

```diff
@@ -0,0 +1,67 @@
+import google.generativeai as genai
+import os
+import subprocess
+import logging
+from dotenv import load_dotenv
+
+
+def get_review(api_key, commit_branch, main_branch, logger):
+    subprocess.run(
+        f"git fetch origin {main_branch} && git fetch origin {commit_branch} && git checkout {commit_branch}",
+        shell=True,
+        check=True,
+    )
+    git_diff = subprocess.run(
+        f"git diff origin/{main_branch}..origin/{commit_branch}",
+        shell=True,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        check=True,
```

Comment:
The code does not handle the case when the git diff command returns an error. It checks if stderr is not empty but does not raise an error. The code should raise an error if stderr is not empty.